intake_bluesky/mongo_embedded.py

```diff
@@ -118,14 +118,14 @@ def get_header_field(field):
                         if field[0:6] == 'count_':
                             return 0
                         else:
-                            return None
+                            return {}
```

Comment:
Under what circumstances would this codepath be hit?

Reply:
example could be. If a run doesn't have a stop document and you ask for the stop document